static/js/utils.js

```diff
@@ -94,21 +93,20 @@ function decimalRaToSexigesimal(deg){
 }
 
 function decimalDecToSexigesimal(deg){
-  var ds = 1;
+  let ds = 1;
+  let dec = deg;
   if(deg < 0){
     ds = -1;
-    var dec = Math.abs(deg);
-  } else {
-    var dec = deg;
+    dec = Math.abs(deg);
   }
-  var deg = Math.floor(dec)
```

Comment:
We had overwritten the deg variable that was passed in before... so im not sure if this equation is correct. We should double check this.

Reply:
This looks fine to me